Attachment_Downloader/token.json

```diff
@@ -0,0 +1 @@
+{"access_token": "ya29.a0AfH6SMCzkE5-xyfKaHz_EKnhKeBvc3oFeWYciu2MxZK70mXy7JbR32l0dnRwNtlWZxyMBaOV10XTkeJqIbWb8z5cUIA2YiiL3KXzt5E41Wx327cZbgU3C0yx1zAtoWlw98V_HjmSOPW7wP8HZlUrcdHKeuzEpA", "client_id": "1004827760520-vhe76mj8bnhl2djbi911hul9ebiuar5p.apps.googleusercontent.com", "client_secret": "jiOeNPFSLKb6TeAGge-Fta-5", "refresh_token": "1//0grs3VlWC1FgpCgYIARAAGBASNwF-L9Ir6d_ihrdvKysfTi86Eqv2w18XTWSxi8x8GHHBoeZjm_viWL3RT7qr45a4KIhYoLMbvd4", "token_expiry": "2021-03-16T13:49:30Z", "token_uri": "https://oauth2.googleapis.com/token", "user_agent": null, "revoke_uri": "https://oauth2.googleapis.com/revoke", "id_token": null, "id_token_jwt": null, "token_response": {"access_token": "ya29.a0AfH6SMCzkE5-xyfKaHz_EKnhKeBvc3oFeWYciu2MxZK70mXy7JbR32l0dnRwNtlWZxyMBaOV10XTkeJqIbWb8z5cUIA2YiiL3KXzt5E41Wx327cZbgU3C0yx1zAtoWlw98V_HjmSOPW7wP8HZlUrcdHKeuzEpA", "expires_in": 3599, "scope": "https://mail.google.com/", "token_type": "Bearer"}, "scopes": ["https://mail.google.com/"], "token_info_uri": "https://oauth2.googleapis.com/tokeninfo", "invalid": false, "_class": "OAuth2Credentials", "_module": "oauth2client.client"}
```

Comment:
And this?

Reply:
> And this?

Token.json file is automatically generated when script runs for the first time. This file prevents user from authenticating again and again.

Reply:
So this should not be uploaded here, it serves no purpose to script

Reply:
Yes will delete this file too. Thanks for pointing out.